kubric/assets/klevr.py

```diff
@@ -14,36 +14,61 @@
 
 import numpy as np
 
+import kubric as kb
 from kubric.assets import asset_source
 from kubric import core
 from kubric import color
 
+KLEVR_ASSETS_IDs = [
+  "LargeMetalCube",
+  "LargeMetalCylinder",
+  "LargeMetalSphere",
+  "LargeRubberCube",
+  "LargeRubberCylinder",
+  "LargeRubberSphere",
+  "MetalSpot",
+  "RubberSpot",
+  "SmallMetalCube",
+  "SmallMetalCylinder",
+  "SmallMetalSphere",
+  "SmallRubberCube",
+  "SmallRubberCylinder",
+  "SmallRubberSphere",
+]
 
 class KLEVR(asset_source.AssetSource):
-  objects_list = [
-      "LargeMetalCube",
-      "LargeMetalCylinder",
-      "LargeMetalSphere",
-      "LargeRubberCube",
-      "LargeRubberCylinder",
-      "LargeRubberSphere",
-      "MetalSpot",
-      "RubberSpot",
-      "SmallMetalCube",
-      "SmallMetalCylinder",
-      "SmallMetalSphere",
-      "SmallRubberCube",
-      "SmallRubberCylinder",
-      "SmallRubberSphere",
-  ]
-
   def __init__(self, uri: str):
     super().__init__(uri)
-    self.spawn_region = ((-3.5, -3.5, 0), (3.5, 3.5, 2))
-    self.ambient_light = (0.05, 0.05, 0.05)
+    #TODO: this seems obsolete?
+    # self.spawn_region = ((-3.5, -3.5, 0), (3.5, 3.5, 2))
+    #TODO: this seems obsolete?
+    # self.ambient_light = (0.05, 0.05, 0.05)
```

Comment:
ambient_light is desired, but currently not used. Probably got lost somewhere along the way. But the interface will change during my refactoring anyway, so I'll have to bring it back afterwards.